### ISSUE TITLE
Individual Illuminate packages are not updated since 5.5.17

### ISSUE THREAD

**Issue by aedart** [laravel/framework#22462]
- Laravel Version: 5.5.17+
- PHP Version:
- Database Driver & Version:

### Description:
Acc. to [packagist](https://packagist.org/packages/illuminate/config), several illuminate packages are still at version `5.5.17`, but the framework itself is now at `5.5.25`. Has something happened in you release process, since the individual illuminate packages are no longer updated?

**Comment by browner12** [comment]
if I remember correctly, this is not an automated process.  I'm guessing the importance of the changes dictates a little how often they are updated.

**Comment by aedart** [comment]
@browner12, okay... but that wouldn't make any sense. Previously, before version `5.5.17`, I received updates regularly, per illuminate package.

I find that this is a very important thing, especially for those of us that develop packages and require just a few illuminate package, instead of the entire framework. Said differently, if Laravel releases important bug-fixes, those appear not to be "pushed" to the individual packages.

**Comment by 36864** [comment]
I'm fairly certain individual packages only get their versions bumped when there are actual changes made to them.

If there's a patch that only affects one package, why should all other packages get a version bump?

More importantly, how does this negatively impact your package and why would things be better if, for example, the config package got version bumped after an unrelated fix to the session package if you're only requiring config and not session?

**Comment by aedart** [comment]
@36864, that sounds very reasonable. If that is the case, then this isn't an issue - however, when reviewed the available versions, I just noticed the version gab and got scared.

UPDATE: Well... I just checked [illuminate/database](https://packagist.org/packages/illuminate/database) - it does not match the version(s) available in the framework, also from `5.5.17`. I read in the release notes for `5.5.25`, which was released 2017-12-11. No way that this NOT a mistake!
